FBSnapshotTestCase/Categories/UIImage+Compare.h

```diff
@@ -34,7 +34,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface UIImage (Compare)
 
-- (BOOL)fb_compareWithImage:(UIImage *)image pixelTolerance:(CGFloat)pixelTolerance tolerance:(CGFloat)tolerance;
+/**
+ Compares the image against another given image.
+
+ @param image The other image to compare against.
+ @param perPixelTolerance How much (in percentage) any given pixel's colors are allowed to change from the pixel in the reference image.
```

Comment:
Having not read the implementation yet, the description for the `perPixelTolerance` parameter left me wondering exactly how the difference is calculated.

After reading the implementation it looks like we compute the change in the red, green, blue and alpha channels and if any of those channels differ by the tolerance amount we count it is changed. Adding something to that effect to this comment would help alleviate the confusion I was having.

Reply:
Yes, I think we need to do something here to make this clearer. Once I figure out what magic number works well for this tolerance for some of the issues we've had, I think I'll understand this more so we can improve the documentation.